### ISSUE TITLE
Filter row cells use `th` elements, while the accessibility spec states they should use `td` instead

### ISSUE THREAD

**Issue by radkostanev** [telerik/kendo-themes#4883]
**Describe the bug**

There is a discrepancy between what the rendering tests suggests when it comes to the filter row cell element and the one provided by our accessibility specs.

To be exact, the [rendering tests](https://github.com/telerik/kendo-themes/blob/develop/tests/grid/grid-filter-row.html#L22) suggest using `<th>` elements, while the [accessibility spec](https://github.com/telerik/web-components-ux/blob/master/docs/components/grid/aria.json#L214) states `<td>` elements should be used despite being part of a `<thead>`.

**To reproduce**

N/A

**Expected behavior**
Unified approach, likely being rendering `<td>` elements.

**Screenshots**
N/A

**Affected package (please remove the unneeded items)**
* theme-default
* theme-bootstrap
* theme-material
* theme-tasks

**Affected suites (please remove the unneeded items)**
* Kendo UI for jQuery
* Kendo UI for Angular
* Kendo UI for React
* Kendo UI for Vue
* Telerik UI for Blazor

**Affected browsers (please remove the unneeded items)**
* Not Applicable

**Build system information (please remove the unneeded items)**
* Not Applicable

**Additional context**

This will likely be addressed by further rendering unification of the table components. However, as multiple ThemeBuilder clients have reported this, we should prioritize it when possible.

**Comment by epetrow** [comment]
@radkostanev, this has been addressed [here](https://github.com/telerik/kendo-themes/pull/4870)

**Comment by radkostanev** [comment]
Great news @epetrow !
